fieldservice/views/fsm_location.xml

```diff
@@ -69,7 +69,7 @@
                         <group id="main-left">
                             <field name="id" invisible="1"/>
                             <field name="fsm_parent_id" domain="[('id', '!=', id)]"/>
-                            <field name="partner_id" groups="base.group_no_one" readonly="1" />
+                            <field name="partner_id" groups="base.group_no_one" readonly="1" required="False" />
```

Comment:
@AmmarOfficewalaSerpentCS : We can't set partner_id to require False. because it's a required field and partner_id will be linked upon location creation itself.

Reply:
@smangukiya , 
we are using a _inherits res partner if partner_id is not set it automatically creates a new partner and set in a related partner.